addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -1568,9 +1568,13 @@ export class Wysiwyg extends Component {
         }
 
         element.classList.toggle('o_we_has_btn_outline_primary',
-            weUtils.getCSSVariableValue('btn-primary-outline') === 'true');
+            weUtils.getCSSVariableValue('btn-primary-outline') === "'True'");
```

Comment:
Why is this becoming `True` with an uppercase letter all of a sudden? Should it be a fix too?

Reply:
Probably a problem with "True" in python. I'll investigate and give you more information about that.

Reply:
So I've investigated this issue. In the method `customizeButtonStyle`, the values passed are real booleans in this case. Theses values are transmitted to the orm through the API call `make_scss_customization` and probably not well parsed. So instead of send `true` / `false`, I send `'true'` / `'false'`. So the CSS variable is correctly set. 

![image](https://github.com/odoo/odoo/assets/22252071/a83bf579-df49-46b8-9f3c-f8a7dc3c7c14)

I've changed the `customizeButtonStyle` to : 
```
async customizeButtonStyle(previewMode, widgetValue, params) {
        await this._customizeWebsiteVariables({
            [`btn-${params.button}-outline`]: widgetValue === 'outline' ? 'true' : 'false',
            [`btn-${params.button}-flat`]: widgetValue === 'flat' ? 'true': 'false',
        }, params.nullValue);
    },
```

Reply:
Ok, this should be fixed in stable versions as well though, no? If so, make a dedicated commit in a stable version PR (where the bug occurs originally). There should probably also be some kind of support for users where the value was already saved as "True" instead of "true"?